docs/secure-quickstart.adoc

```diff
@@ -0,0 +1,99 @@
+= Beer Garden in 7 minutes
+:page-layout: docs
+:plugin-include-dir: python/_includes
+:install-include-dir: installation-guides/_includes
+:plugin-base-image: bgio/plugins
+
+
+Get up and running with Beer Garden securely!
+
+This guide is designed to provide a secure alternative to the Beer-Garden installation from the Quickstart. If you haven't gone through the Quickstart go ahead and run though that first.
+
+
+== Out with the old
+
+Before we can bring up a secure Beer-Garden we need to bring down the non-secure one:
+
+[source]
+----
+cd beer-garden/docker/docker-compose
+
+docker-compose down
+----
+
+
+== Place your certs
+
+In order to operate securely you MUST be using TLS, which means you'll need a certificate. How to obtain a certificate is outside the scope of this tutorial, but a good place to start is https://letsencrypt.org[Let's Encrypt].
+
+In the `beer-garden/docker/docker-compose/data` directory there's a `certs` directory with several files. These need to be overwritten with your specific files:
+
+- server_key.pem: The private portion of the server certificate
+- server_certificate.pem: The public portion of the server certificate
+- ca_certificate.pem: If your certificate is issued by a non-standard Certificate Authority you'll need to replace this. Otherwise, remove it completely.
+
+
+== Update the compose
+
+To operate in secure mode we need to use a slightly different docker-compose file. The file comes with several defaults that need to be changed in order to be acutally secure, so we need to do that next. Open the docker-compose.yml file that's in the `tls` directory.
+
+Your TLS certificate will be issued for a specific host name. That name needs to be set for these three items (indicated with a `# Replace` comment in the compose file):
+
+- BG_PUBLISH_HOSTNAME
+- BG_AMQ_HOST
+- BG_WEB_HOST
+
+Connecting to RabbitMQ is controlled by a username and password. These have default values in the compose file that need to be changed. Note that the same user/password combination should be used for all three entries:
```

Comment:
Is that true that we need these to be the same?

Also, if we are going to call this one the secure quickstart, we should highly encourage them to change these values.